packages/mobile/src/send/SendAmount.tsx

```diff
@@ -353,14 +353,16 @@ const styles = StyleSheet.create({
   },
   contentContainer: {
     flex: 1,
+    width: Dimensions.get("window").width,
   },
   showAmountContainer: {
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'center',
+   flex: 1,
+   flexDirection: 'row',
+   justifyContent: 'center',
   },
   amountContainer: {
     justifyContent: 'center',
+    maxWidth: 256,
```

Comment:
This shouldn't be absolute, so it adapts better to different screen widths.
What about `'90%'`? instead?

Reply:
for sure! ill try it out. makes sense